File: w11/d03/Classwork/fakepeople.py
from faker import Faker
fake = Faker()

people = {}

class Person:
	def __init__(self, name, phone_number):
		self.phone_number = phone_number
		self.name = name

for i in range(1,1001):
	people[i] = {Person(fake.name(), fake.phone_number())}

print people

File: w11/d03/Homework/fakepeople.rb
require 'faker'
require 'pry'
people = {}

class Person
	def initialize(info)
		@name = info[:name]
		@phone_number = info[:phone_number]
	end
end

(1..1000).each do |person|
	new_person = Person.new({name: Faker::Name.name , phone_number: Faker::PhoneNumber.cell_phone})
	people[person] = new_person
end

print people

binding.pry